mnemonic-core/src/test/java/org/apache/mnemonic/DurablePersonNGTest.java

```diff
@@ -78,25 +80,74 @@ public boolean reclaim(Long mres, Long sz) {
         }
 
         person = PersonFactory.create(act);
+        int size = rand.nextInt(1024 * 1024) + 1024 * 1024;
+/////////////////////////if size is smaller than remaining act capacity////
+
+        mbh = act.createBuffer(size);
+        if (mbh != null) {
```

Comment:
Please throw OutOfHybridMemory if no memory space to allocate and also for others